Applications/WebPortal/WebApp/library/source/pages/account_page/compliance_page/new_rule_button.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { DisplaySize } from '../../../display_size';
+
+interface Properties {
+  
+  /** The size at which the component should be displayed at. */
+  displaySize: DisplaySize;
+}
+
+export class NewRuleButton extends React.Component<Properties> {
+  constructor(props: Properties) {
```

Comment:
Since this constructor only calls its class' parent constructor, you can remove it entirely. The default constructor will do this anyways.